src/site/content/en/animations/animations-overview/index.md

```diff
@@ -58,22 +63,23 @@ changes further down the tree can sometimes result in layout calculations all th
 The larger the tree of visible elements,
 the longer it takes to perform layout calculations.
 
-## Animating paint properties
+### Animating paint properties {: #paint }
 
-If layout has been triggered then the next step is to move onto paint.
-In addition, animating some properties may allow the browser to skip the layout step and start with paint.
+[Paint](https://developers.google.com/web/updates/2018/09/inside-browser-part3#paint)
+is the process of determining in what order elements should be painted to the screen.
+It is often the longest-running of all tasks in the pipeline.
```

Comment:
For the first sentence of each section I tried elaborating on the meaning of each stage of the pipeline. Not sure if this is the best approach.